packages/leaders-program/src/graphql/queries/content-for-section.js

```diff
@@ -107,7 +108,7 @@ query ContentForLeadersSection(
             channelId
             url
           }
-          videos: youtubeVideos(input: { pagination: { limit: $videoLimit } }) {
+          videos: youtubeVideos(input: { pagination: { limit: $videoLimit }, showPrivate: $showPrivate }) {
```

Comment:
You need to actually define the `showPrivate` input on the `ContentCompany.youtubeVideos` resolver. If you were to pass true to this input the GQL request would error because `showPrivate` isn't actually defined.

Also, is there a case in the leaders package where you're actually passing this in as `true`? I don't see that anywhere, so there really isn't a need to added this to the query.

That said, you _must_ add the input definition to the graphql server (as mentioned above).

Everything in GraphQL must be defined (types, inputs, enums, etc etc)... it does not allow for arbitrary values